docs/variants/msi_z690/cpu-hcl.md

```diff
@@ -2,41 +2,158 @@
+    FSP (Firmware Support Package) component is initializing the Memory Modules. We
+    don't stress test them for stability. Each SPD Profile is tested, so each
+    Memory Module can have from 1 to 4 entries. You may also check out
+    [Intel memory validation results](https://www.intel.com/content/www/us/en/platform-memory/platform-memory.html)
+    page and see the results for DDR4 UDIMM testing. Maximum ratings for Alder Lake
+    S CPUs is also available in the [CPU datasheet](https://edc.intel.com/content/www/us/en/design/ipla/software-development-platforms/client/platforms/alder-lake-desktop/12th-generation-intel-core-processors-datasheet-volume-1-of-2/001/processor-sku-support-matrix/).
+
+    Legend:
+
+    * Configuration 1/2/4 - means given memory module was tested in 1, 2 and 4
+      DIMMs populated configuration, &#10004; means successfully tested, &#10006;
+      means platform did not boot with Dasharo, e.g. &#10004;/&#10004;/&#10004;
+      means all configurations work, `-` means not tested
+    * Size: DIMM capacity in GB
+    * SPD profile: can be one of JEDEC(Standard) / XMP Profile #1 / XMP Profile #2.
+      Profile data:
+        - Type/speed: for example DDR4-2400 means DDR4 module clocked at max 2400MHz
+        for given profile
+        - Timings: for example CL17-17-17 means CAS Latency 17, tRCD 17, tRP 17
+        (numbers expressed in clock cycles) for given memory profile
+        - Voltage: memory voltage in Volts for given memory profile
+
+    > NOTE: some XMP profiles may have lower speeds than other ones, but also have
+    > smaller CAS latency. Also the memory frequency configured in the memory
+    > controller and reported by DMI/SMBIOS may be slightly lower than advertised
+    > in the module specifications. This can vary based on the board design, CPU
+    > and many other factors.
+
+    ### HCL list - MSI PRO Z690-A DDR4
+
+    | Manufacturer | Part Number | Size | Type | Speed | Configuration 1/2/4 | Dasharo version | Source |
+    |:------------:|:-----------:|:----:|:----:|:-----:|:-------------------:|:---------------:|:------:|
+    | Crucial Technology | CT8G4DFS8266.C8FJ | 8192 MB | DDR4 SDRAM | 2666 MT/s (PC4-21300) | &#10004/-/- | v1.1.0 | Dasharo HCL report |
+    | Kingston | KF3600C17D4/8GX      | 8192 MB | DDR4 SDRAM | 2400 MT/s (PC4-19200) | &#10004/-/&#10004 | v1.0.0 | Dasharo HCL report |
+    | Kingston | KF3600C17D4/8GX      | 8192 MB | DDR4 SDRAM | 2400 MT/s (PC4-19200) | -/-/&#10004 | v1.1.0 | Dasharo HCL report |
+    | Patriot Memory | 4400 C19 Series | 8192 MB | DDR4 SDRAM | 2133 MT/s (PC4-17000) | &#10004/-/- | v1.0.0 | Dasharo HCL report |
+    | Thermaltake Technology Co Ltd | RG26D408GX2-3600C18A | 8192 MB | DDR4 SDRAM | 2666 MT/s (PC4-21300) | &#10004/-/- | v1.1.0 | Dasharo HCL report |
+
+    1) Tested in 2x2 mixed configuration
+
+### HCL list - MSI PRO Z690-A DDR5
+
+| DIMM vendor | Part Number | Size | SPD profile | Configuration 1/2/4 |
```

Comment:
This table is less complete than the previous one

Reply:
@macpijan it is; the whole work still needs to be completed. It is more of RFC; before investing too much time, I wanted to get your feedback about the change in approach. Please note that it took us three months to check this PR - it isn't efficient. Let's say I will redesign the whole concept of reporting all components. It still has to align with all other efforts of DTS and TAT otherwise, we keep having 3 competing solutions.  I had hoped for some practical cross-team cooperation.

Reply:
@macpijan it looks like we don't have HCL reports dumps for DDR5 because those were not generated during testing. So either we leave this table as is, despite we have no proof in the form of logs or we remove it.